file-formats/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -9,14 +9,27 @@ INTERFACE zif_aff_cota_v1
     "! <p class="shorttext">Communication Type</p>
     "! Communication type
     BEGIN OF co_comm_type,
-      "! <p class="shorttext">RFC</p>
-      "! RFC
-      rfc  TYPE ty_comm_type VALUE 'R',
       "! <p class="shorttext">HTTP</p>
       "! HTTP
       http TYPE ty_comm_type VALUE 'H',
     END OF co_comm_type.
 
+  "! <p class="shorttext">Multitenancy Mode</p>
+  "! $values {@link zif_aff_cota_v1.data:co_multimode_type}
+  TYPES ty_multi_type TYPE c LENGTH 30.
```

Comment:
You can align to nomenclature above where you have `co_comm_type` and `ty_comm_type` here as well.

Reply:
here its `ty_multi_type` and `co_multimode_type`

Reply:
I am planning to adopt :)